### PR TITLE
Updating copy throughout the site based on Kate's feedback

### PR DESCRIPTION
**Pull request by noahmanger**
Simple copy edits throughout the site to clean up the explanatory language. There will still be small things down the road, but this addresses what I knew to be issues. 

We still need go through and clean up / add additional glossary definitions with the FEC's help (created a [new ticket for this](https://github.com/18F/openFEC/issues/576) and also [add the correct links](https://github.com/18F/openFEC/issues/577). 

But let's get this in front of them to review for now. 

Addresses: https://github.com/18F/openFEC/issues/552

### CODE REVIEW

File: templates/partials/financial-summary.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
   {% if related_committees %}
       <div class="page-section__intro">
         <h2 id="section-1-header" tabindex="0">Financial Summary</h2>
-        <p class="text--lead">Get the full picture of all of the money received and spent by this candidate’s principal and authorized committees.</p>
+        <p class="text--lead">This page provides an overview of all the money received and spent by the committees associated with this candidate.</p>
+        <p class="text--lead">Candidates are directly responsible for their <strong>principal campaign committees</strong> and <strong>authorized campaign committees</strong>. Candidates are associated with other types like <strong>leadership PACs</strong> and <strong>joint fundraising committees</strong>, but because those committees are not exclusively for the candidate's campaign activity, this page will not include their information.</p>
```
Comment (LindsayYoung) [inline templates/partials/financial-summary.html]: Joint committees are on this page and I would love to have leadership pacs on this page in the future.

Comment (noahmanger) [inline templates/partials/financial-summary.html]: Ah ok. I'll remove leadership pacs.

File: templates/search.html [hunk 2]
```diff
@@ -6,8 +6,8 @@
     <div class="container">
         <div class="page-section__intro">
             <h2 id="browse-heading">Get to Know the Data</h2>
-            <p class="text--lead">Not sure exactly what you’re looking for? Start with a full set of data and then filter down to what you’re looking for.</p>
-            <p class="text--lead">We currently have data for all candidates, committees, and their reports and filings.</p>
+            <p class="text--lead">Want to learn more about federal election data but aren’t sure exactly what you’re looking for? Start with a full set of data, then apply specific filters to find the information most relevant to you.</p>
+            <p class="text--lead">OpenFEC currently has financial data for all candidates for federal office and all committees involved in federal races, and we will be adding more soon.</p>
```
Comment (LindsayYoung) [inline templates/search.html]: We don' have finical data for all types of committees. We do have their profile information. Though, maybe that is not a distinction that matters to most people.

Comment (noahmanger) [inline templates/search.html]: I think it's fine for now?

On Fri, Apr 17, 2015 at 11:23 AM, Lindsay Young <notifications@github.com>
wrote:

> In templates/search.html
> <https://github.com/18F/openFEC-web-app/pull/75#discussion_r28603418>:
>
> > @@ -6,8 +6,8 @@
> >      <div class="container">
> >          <div class="page-section__intro">
> >              <h2 id="browse-heading">Get to Know the Data</h2>
> > -            <p class="text--lead">Not sure exactly what you’re looking for? Start with a full set of data and then filter down to what you’re looking for.</p>
> > -            <p class="text--lead">We currently have data for all candidates, committees, and their reports and filings.</p>
> > +            <p class="text--lead">Want to learn more about federal election data but aren’t sure exactly what you’re looking for? Start with a full set of data, then apply specific filters to find the information most relevant to you.</p>
> > +            <p class="text--lead">OpenFEC currently has financial data for all candidates for federal office and all committees involved in federal races, and we will be adding more soon.</p>
>
> We don' have finical data for all types of committees. We do have their
> profile information. Though, maybe that is not a distinction that matters
> to most people.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/openFEC-web-app/pull/75/files#r28603418>.
>



-- 
Noah Manger
18F <http://18f.gsa.gov> | Work: 415-696-6146

Comment (noahmanger) [inline templates/search.html]: Nevermind, I edited.

**Comment by rjmajma** [comment]
We good here? @LindsayYoung if Noah squared away your comments would you mind merging?

**Pull request merged**